org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -28,6 +28,11 @@
 	public static final String REFERENCES_CODE_LENS_ENABLED_KEY = "java.referencesCodeLens.enabled";
 
 	/**
+	 * Preference key to enable/disable formatter.
+	 */
+	public static final String JAVA_FORMAT_ENABLE_KEY = "java.format.enable";
```

Comment:
so I think we need to be consistent with ourselves here, even if we break consistency with vscode and its `javascript.referencesCodeLens.enabled` vs `javascript.format.enable` pref keys. So I'd use `java.format.enabled` instead